hera_cal/utils.py

```diff
@@ -530,3 +654,52 @@ def lst_rephase(data, bls, freqs, dlst, lat=-30.72152, inplace=True, array=False
         
     if inplace == False:
         return data
+
+
+def data_to_gain_flags(flags, gain_keys=None):
```

Comment:
This function already exists in smooth_cal: https://github.com/HERA-Team/hera_cal/blob/6085944949e0ab21a85093989754454b8b6d055b/hera_cal/smooth_cal.py#L23

Let's figure out if either version is better and then keep that one. I think utils is a fine place for it.

Reply:
ah didn't see that, thanks for pointing it out. looks like yours accomplishes what I need, so i'm happy to just use that. can you confirm that your function is doing the same thing as mine is here?